models/campaign.go

```diff
@@ -157,6 +158,26 @@ func (c *Campaign) UpdateStatus(s string) error {
 func (c *Campaign) AddEvent(e *Event) error {
 	e.CampaignId = c.Id
 	e.Time = time.Now().UTC()
+
+  //INFO send webhook
+  //TODO this code is duplicated at models/result.go#sendWebhooks
+  whs, err := GetActiveWebhooks()
+  if err != nil {
+		whEndPoints := []webhook.EndPoint{}
+	  for _, wh := range whs {
+			whEndPoints = append(whEndPoints, webhook.EndPoint {
+			  Url: wh.Url,
+			  Secret: wh.Secret,
+			})
+	  }
+	  pl := map[string]interface{} {
```

Comment:
As mentioned before, let's please just send the event as is. This is still wrapping it in a struct, which isn't necessary. In this case, we should just run `webhook.SendAll(whEndpoints, e)`. This will subsequently marshal the event into JSON and send it out, so what's received by the webhook would essentially be the same thing that's returned via the API.

Reply:
done

Reply:
This doesn't appear done - it looks like you're still sending `pl` to `SendAll`.